ta/avb/entry.c

```diff
@@ -19,28 +23,50 @@ static TEE_Result get_slot_offset(size_t slot, size_t *offset)
 	return TEE_SUCCESS;
 }
 
-static TEE_Result create_state(uint32_t lock_state, TEE_ObjectHandle *h)
+static TEE_Result create_rb_state(uint32_t lock_state, TEE_ObjectHandle *h)
 {
 	const uint32_t flags = TEE_DATA_FLAG_ACCESS_READ |
 			       TEE_DATA_FLAG_ACCESS_WRITE |
 			       TEE_DATA_FLAG_OVERWRITE;
 
-	return TEE_CreatePersistentObject(storageid, obj_name, sizeof(obj_name),
-					  flags, NULL, &lock_state,
-					  sizeof(lock_state), h);
+	return TEE_CreatePersistentObject(storageid, rb_obj_name,
+					  sizeof(rb_obj_name), flags, NULL,
+					  &lock_state, sizeof(lock_state), h);
 }
 
-static TEE_Result open_state(uint32_t default_lock_state, TEE_ObjectHandle *h)
+static TEE_Result open_rb_state(uint32_t default_lock_state,
+				TEE_ObjectHandle *h)
 {
-	uint32_t flags = TEE_DATA_FLAG_ACCESS_READ | TEE_DATA_FLAG_ACCESS_WRITE;
+	uint32_t flags = TEE_DATA_FLAG_ACCESS_READ |
+			 TEE_DATA_FLAG_ACCESS_WRITE;
 	TEE_Result res;
 
-	res = TEE_OpenPersistentObject(storageid, obj_name,
-				       sizeof(obj_name), flags, h);
+	res = TEE_OpenPersistentObject(storageid, rb_obj_name,
+				       sizeof(rb_obj_name), flags, h);
 	if (!res)
 		return TEE_SUCCESS;
 
-	return create_state(default_lock_state, h);
+	return create_rb_state(default_lock_state, h);
+}
+
+static TEE_Result get_named_object_name(char *name_orig,
+					uint32_t name_orig_size,
+					char *name, uint32_t *name_size)
+{
+	if (name_orig_size + strlen(named_value_prefix) >
```

Comment:
Saving the result of `strlen(named_value_prefix)` in a variable with a short name would simplify the code a bit.